src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -23,6 +23,10 @@
     /** The directory where JSON files used to create data bundles are stored. */
     public static final String TEST_DATA_FOLDER = "src/test/resources/data";
 
+    /** The directory where credentials used in Gmail API are stored. */
+    @SuppressWarnings("WeakerAccess")
```

Comment:
I feel the suppression should be removed as we have plenty of warnings all over the place, and the question of which warnings show up where is pretty IDE-specific (and can even be configured in the IDE).